src/plugins/myItems/myItemsInterceptor.js

```diff
@@ -20,24 +20,31 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-export default function MyItemsInterceptor(openmct) {
+import { MY_ITEMS_KEY } from "./createMyItemsIdentifier";
+const MISSING_NAME = `Missing: ${MY_ITEMS_KEY}`;
 
-    openmct.objects.addGetInterceptor({
-        appliesTo: (identifier, domainObject) => {
-            return identifier.key === 'mine';
+function myItemsInterceptor(identifierObject) {
+
+    const myItemsModel = {
+        identifier: identifierObject,
+        "name": "My Items",
+        "type": "folder",
+        "composition": [],
+        "location": "ROOT"
+    };
+
+    return {
+        appliesTo: (identifier) => {
+            return identifier.key === MY_ITEMS_KEY;
         },
         invoke: (identifier, object) => {
-            if (object === undefined) {
-                return {
-                    identifier,
-                    "name": "My Items",
-                    "type": "folder",
-                    "composition": [],
-                    "location": "ROOT"
-                };
+            if (object === undefined || object && object.name === MISSING_NAME) {
```

Comment:
Hrm. Do we still get undefined objects at this point? The "my items" object should be a missing object at this point due to the change in https://github.com/nasa/openmct/pull/4107/files#diff-0ea3f7f0c3792d3090ca667730894b2c328be4cf58596695f8a649d7f4dca54aR26 no?

Reply:
Also, we should add an Object API method for testing whether an object is a missing object

Reply:
There should be no inconsistency in the order in which the object interceptors are called, so if you've tested it and it works then it should continue to work. 

The problem is that we are relying on implicit ordering, which is determined by the order that plugins are registered. We should add an explicit priority to our object interceptors. I'll add an issue to do this.

Reply:
Removed!